apps/condo/domains/organization/integrations/sbbol/routes.js

```diff
@@ -81,7 +84,7 @@ class SbbolRoutes {
             const { finished } = await getOnBoardingStatus(user)
 
             logger.info({ msg: 'SBBOL OK Authenticated', userId: user.id, organizationId: organization.id, employeeId: organizationEmployee.id, data: { finished } })
-
+            if (redirectUrl && isSafeUrl(redirectUrl)) return res.redirect(redirectUrl)
```

Comment:
Check for valid `redirectUrl` should be performed in `startAuth` to avoid saving wrong value to session and to provide feedback as earlier, as possible.